app/src/main/java/com/dsd/carcompanion/VehicleInfoAdapter.kt

```diff
@@ -0,0 +1,30 @@
+package com.dsd.carcompanion
```

Comment:
This file should go in its own folder named like adapters

Reply:
done